cmd/create/accountroles/cmd.go

```diff
@@ -412,5 +438,25 @@ func createRoles(r *rosa.Runtime, prefix, permissionsBoundary, accountID, env st
 		}
 	}
 
+	if confirm.Prompt(true, "Create the operator policies?") {
+		for credrequest, operator := range credRequests {
+			policyARN := aws.GetOperatorPolicyARN(accountID, prefix, operator.Namespace(), operator.Name(), rolePath)
+			filename := fmt.Sprintf("openshift_%s_policy", credrequest)
+			policyDetails := policies[filename]
+
+			policyARN, err := r.AWSClient.EnsurePolicy(policyARN, policyDetails,
+				defaultPolicyVersion, map[string]string{
+					tags.OpenShiftVersion: defaultPolicyVersion,
+					tags.RolePrefix:       prefix,
+					"operator_namespace":  operator.Namespace(),
+					"operator_name":       operator.Name(),
+				}, policyPath)
```

Comment:
@pvasant @ciaranRoche Could you please do a quick sanity check whether the `policyPath` is the right value for the last arg here?